src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralConfiguredBrowseObject.xaml

```diff
@@ -43,9 +43,14 @@
   <!-- Advanced Build Page Properties -->
   <EnumProperty Name="LanguageVersion" DisplayName="Language version" Visible="False">
     <EnumProperty.DataSource>
-      <DataSource Persistence="ProjectFile" PersistedName="LangVersion" SourceOfDefaultValue="AfterContext"/>
+      <DataSource Persistence="ProjectFile" PersistedName="LangVersion" SourceOfDefaultValue="AfterContext" HasConfigurationCondition="False" />
     </EnumProperty.DataSource>
   </EnumProperty>
+  <StringProperty Name="LangVersion" DisplayName="Language version" Visible="False" >
```

Comment:
Why do we have both `LangVersion` and `LanguageVersion?`

Reply:
> I have no idea why we have two properties here; one for consumed by CSharpProjectConfigurationProperties and one consumed by AppDesigner but I left the exercise for a latter time.

Reply:
Removing either breaks one of the features.